ReactiveSwift.xcodeproj/project.pbxproj

```diff
@@ -1512,6 +1512,7 @@
 			isa = XCBuildConfiguration;
 			baseConfigurationReference = D047262919E49FE8006002AA /* Debug.xcconfig */;
 			buildSettings = {
+				APPLICATION_EXTENSION_API_ONLY = YES;
```

Comment:
Which target did you set this on? It's already `YES` on each of the 4 targets.
![Screenshot 2023-01-05 at 10 13 16](https://user-images.githubusercontent.com/685609/210851001-712b2878-5584-4011-b8a3-b7aeb5437ae4.png)

Reply:
Oh sorry, I didn't have the XCConfig submodule setup correctly. I've reverted this.